doc/man7/SM2.pod

```diff
@@ -41,7 +41,7 @@ done by calling:
 And normally there is no need to pass a B<pctx> parameter to EVP_DigestSignInit()
 or EVP_DigestVerifyInit() in such a scenario.
 
-SM2 can be tested within L<speed(1)> application since version 3.0.0. At current
+SM2 can be tested within L<openssl-speed(1)> application since version 3.0.0. At current
```

Comment:
within **the** L<openssl-speed(1)> application

_(and break the line after 3.0.0. ?)_

Reply:
fixed, broke line, minor other wording change.